workflows/kfdrc-pacbio-longreads-workflow.cwl

```diff
@@ -160,43 +168,54 @@ steps:
       output_file_name:
         source: output_basename
         valueFrom: $(self).dnascope.vcf.gz
-      cpu_per_job: dnascope_cores
+      cpu_per_job: dnascope_cpu
       mem_per_job: dnascope_ram
     out: [output_vcf]
-
   pbsv_discover:
+    hints:
+    - class: "sbg:AWSInstanceType"
+      value: c5.9xlarge
     run: ../tools/pbsv_discover.cwl
     in:
       input_bam: pbmm2_align/output_bam
       output_filename:
         source: output_basename
         valueFrom: $(self).pbsv.svsig.gz
-      hifi_preset: pbsv_hifi_preset
-      cores: pbsv_cores
+      hifi_preset:
+        source: pbmm2_preset
+        valueFrom: |
+          $(self == "HIFI" || self == "CCS")
```

Comment:
So this is meant to be a boolean...

Reply:
Pretty much. If the user chooses "HIFI" or "CCS" for `pbmm2_preset` this statement will return true and activate the boolean option.